### ISSUE TITLE
Which repo to follow?

### ISSUE THREAD

**Issue by dolmen** [daxim/Module-CPANTS-Analyse#2]
https://github.com/domm/CPANTS also exists.
Which one should I follow/fork?

See also https://github.com/domm/CPANTS/issues/9
Cc: @chorny @domm @charsbar

**Comment by daxim** [comment]
Metafile resources:

```
"bugtracker" : {
    "web" : "https://rt.cpan.org/Dist/Display.html?Name=Module-CPANTS-Analyse"
},
"repository" : {
    "url" : "https://github.com/daxim/Module-CPANTS-Analyse"
}
```

**Comment by dolmen** [comment]
I'm asking because the lastest CPAN release (0.85) says: https://github.com/domm/CPANTS
See https://metacpan.org/module/Module::CPANTS::Analyse

And also because some commits by @rurban occured after the fork to make a 0.86 release. And there is on the CPAN a 0.86 dev release dated 2012-04-10 that may not be that one.

**Comment by daxim** [comment]
Wrong on both counts. 0.86-TRIAL-2 ist the latest CPAN release, the metacpan page you linked to says clearly so. rurban never released a 0.86, lacking PAUSE permission. His patches have been integrated long ago:
- domm/CPANTS@e4af55b95de55100988a538adea99b5e072e2e8c was rebased as daxim/Module-CPANTS-Analyse@c5189da06c8733d3c61b01768b6dfa86782fbacb
- domm/CPANTS@66cbf2ee736882e3b478b8da20f9ddfb630a6d2a was rebased as daxim/Module-CPANTS-Analyse@0c09b5732dbd34dddb69f51d67554a97135972bf
- domm/CPANTS@3caa00251a893ce23e56fa5a2828fe94869672f2 is moot, was already patched in daxim/Module-CPANTS-Analyse@41d6746aae671299cef7f3dc87aa3371787acab1
- domm/CPANTS@d4324220fd3f2ed456554ec08b602cfd6d206fd7 is moot, was already patched in daxim/Module-CPANTS-Analyse@8873046a0395ba1b2c3d650cfc5a0e5916ef3a9c
